x-pack/plugins/security_solution/public/resolver/models/time_range.test.ts

```diff
@@ -9,14 +9,13 @@ import { maxDate, createRange } from './time_range';
 describe('range', () => {
   it('creates a range starting from 1970-01-01T00:00:00.000Z to +275760-09-13T00:00:00.000Z by default', () => {
     const { from, to } = createRange();
-    expect(from.toISOString()).toBe('1970-01-01T00:00:00.000Z');
-    expect(to.toISOString()).toBe('+275760-09-13T00:00:00.000Z');
+    expect(from).toBe('1970-01-01T00:00:00.000Z');
```

Comment:
nit: I guess once we make the backend's time range optional we can just remove this test file and the model below. I think we're only using the `maxDate` exported variable anyway.